cmd/analyse.go

```diff
@@ -65,13 +67,19 @@ func analyseResults(cmd *cobra.Command, ts *suite.TestSuite, results []result.Ne
 
 	log.Println("")
 
-	// nolint
-	op, _ := cmd.Flags().GetString("operation")
-	// nolint
-	hostname, _ := cmd.Flags().GetString("hostname")
+	//nolint
+	//	op, _ := cmd.Flags().GetString("operation")
+	op := ""
```

Comment:
Why are these flags not reading from viper any more?

Reply:
Sorry, I'll change that now -- I had problems with the flags earlier but forgot to change it back. Will fix for next commit